js/qmwp-search-correlations.js

```diff
@@ -13,6 +13,16 @@ var quantimodoSearch = angular.module('quantimodoSearch', ['ui.bootstrap']);
 
 // Define config
 quantimodoSearch.config(function ($httpProvider) {
+    console.log($httpProvider.defaults.headers.common);
+
+    if (typeof mashapeKey !== 'undefined' && mashapeKey) {
+        $httpProvider.defaults.headers.common['X-Mashape-Key'] = mashapeKey;
+    }
+
+    if (typeof accessToken !== 'undefined' && accessToken) {
+        $httpProvider.defaults.headers.common['Authorization'] = "Bearer " + accessToken;
```

Comment:
Line is too long.<br>['Authorization'] is better written in dot notation.<br>Mixed double and single quotes.<br>'accessToken' is not defined.